### DISCUSSION TITLE
Bounces url is not sending data back

### DISCUSSION THREAD

**Discussion by Chris02468** [postalserver/postal#2731] [Help with using Postal]
I have a mailwizz instance that is linked to postal.  I have setup the webhook for bounces using the webhook url from mailwizz in the setup process however it is not sending the bounce data back to mailwizz but I can see it on the postal side, Where do I start in trouble shootings this

**Comment by sandrovenezuela** [comment]
You need to use the "MessageDeliveryFailed" option.

I don't know why, but the "MessageBounce" option is never used to signal any kind of problem with the email.
